tests/integration/components/list-filter-test.js

```diff
@@ -61,12 +61,11 @@ module('Integration | Component | list-filter', function(hooks) {
       </ListFilter>
     `);
 
-
+    await fillIn(this.element.querySelector('.list-filter input'),'s');
     await triggerKeyEvent(this.element.querySelector('.list-filter input'), "keyup", 83);
+    await settled();
```

Comment:
No need for settled here, `await triggerKeyEvent` automatically is waiting for settledness...

Reply:
Please note I did not add the use of the `settled` helper, I simply am updating it to use await in order to be more consistent with other asynchronous test helpers. If I remove `settled`, the documentation for the tutorial will need to be updated as well, and so far I am not getting an answer on whether or not that's a good idea because I believe this is the only time `settled` is introduced in the tutorial. https://github.com/ember-learn/guides-source/pull/390

Reply:
Yep, I totally understand! Would you mind creating an issue over on guides-source that we can use to track my concern here?

I think the rough checklist for that is:

* Remove usage of `settled` after another async helper (my comments here)
  * in guides-source
  * in super-rentals
* Add another example of using `settled` to the tutorial

Reply:
Updated example and PR'd guides source